src/test/java/nl/tudelft/pixelperfect/GameSmokeTest.java

```diff
@@ -0,0 +1,50 @@
+package nl.tudelft.pixelperfect;
+
+import java.awt.AWTException;
+import java.awt.Robot;
+import java.awt.event.KeyEvent;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Test Suite for the Game class.
+ * 
+ * @author Jesse Tilro
+ *
+ */
+
+@SuppressWarnings("PMD")
+public class GameSmokeTest {
+
+  /**
+   * Increase the test coverage by running the Game class methods inside a test case. There is
+   * little functionality to test here, just integration with the jMonkeyEngine.
+   * 
+   * Added the nice threading in order to be able to kill the application.
+   * 
+   * @throws InterruptedException
+   *           Since we're sleeping in this test.
+   * @throws AWTException
+   *           Since we're using the Robot.
+   */
+  @Ignore
```

Comment:
As a side note, this test suite is ignored until Jesse and I can figure out problems with our smoke tests.